src/lib.rs

```diff
@@ -132,23 +132,15 @@
 //! * structure!() macro takes a literal string as an argument.
 //! * It's called `structure` because `struct` is a reserved keyword in Rust.
 
-#[macro_use]
-extern crate proc_macro_hack;
 
 #[doc(hidden)]
-pub extern crate byteorder;
+pub extern crate core2;
```

Comment:
This does not need to be `pub`, and `extern crate` itself is unnecessary -> can be removed.

Reply:
This has been resolved on my branch.